.cspell/custom-words.txt

```diff
@@ -88,6 +88,7 @@ duckpy
 dvipdfmx
 egldeviceintegrations
 EINVAL
+endup
```

Comment:
`endup` doesn't exit, should be written `end-up` or `end up`

Reply:
![image](https://github.com/Scille/parsec-cloud/assets/3187637/37baced1-b557-4966-9c85-c92a117029c7)

Reply:
done.